Fix pkg_resources error message

I've spent a load of time trying to figure out this error:

```
dependency.py:6:1: error: Library stubs not installed
for "pkg_resources"  [import-untyped]
    import pkg_resources
    ^
dependency.py:6:1: note: Hint: "python3 -m pip install types-setuptools"
dependency.py:6:1: note: (or run "mypy --install-types" to install all missing stub packages)
dependency.py:6:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 17 source files)
```

The error would not go away by following the suggestion. It seems that because setuptools was already installed, the stubs were ignored, thus repeating the error.

The correct solution was to upgrade setuptools to a more recent release.

--- a/mypy/stubinfo.py
+++ b/mypy/stubinfo.py
@@ -47,7 +47,6 @@ legacy_bundled_packages: dict[str, str] = {
     "mock": "types-mock",
     "OpenSSL": "types-pyOpenSSL",
     "paramiko": "types-paramiko",
-    "pkg_resources": "types-setuptools",
     "polib": "types-polib",
     "pycurl": "types-pycurl",
     "pymysql": "types-PyMySQL",

